src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -41,7 +42,11 @@ public class TelegramBotClient : ITelegramBotClient
 
         private static readonly JsonSerializerSettings SerializerSettings = new JsonSerializerSettings
         {
-            NullValueHandling = NullValueHandling.Ignore
+            NullValueHandling = NullValueHandling.Ignore,
+            ContractResolver = new DefaultContractResolver
```

Comment:
I know request serialization are only in library but I'd prefer to create a set of reusable request classes that could potentially be used outside of Bot API and don't rely on specific serialization settings to be provided.